app/controllers/company/opening.js

```diff
@@ -41,7 +41,25 @@ function newOpening(req, res, next) {
     res.status(200).json(fetchedInstance);
   });
 }
-
+/**
+ * Function to update opening Details
+ * @param {Object} req Request Object
+ * @param {Object} res Response Object
+ * @param {Function} next Function to pass control to the next middleware
+ */
+function updateOpeningDetails(req, res, next) {
+  openingLib.updateJobOpening(req.user, req.body, function(
```

Comment:
`req.user` passes object to the lib file, we only require company id to be passed